### PR TITLE
Session data cleanup for session_app_info, session_meta_data tables

### PR DESCRIPTION
**Pull request by DinikaSen**
This PR
-  Introduces session data clean up for the IDN_AUTH_SESSION_APP_INFO_TABLE, IDN_AUTH_SESSION_META_DATA tables to the existing mysql, postgresql-9, postgresql-11, mssql, oracle session data scripts.
- Adds cleaning up of IDN_AUTH_USER_SESSION_MAPPING table to the operational data clean up section of all scripts.
- Adds a check to verify if the table exists before cleaning up the tables - IDN_AUTH_USER_SESSION_MAPPING, IDN_AUTH_SESSION_APP_INFO_TABLE, IDN_AUTH_SESSION_META_DATA

Resolves : https://github.com/wso2/product-is/issues/9929, https://github.com/wso2/product-is/issues/5287, https://github.com/wso2/product-is/issues/7996

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/oracle/sessiondata-cleanup/oracle-sessiondata-cleanup.sql [hunk 1]
```diff
@@ -127,14 +137,49 @@ BEGIN
...
+            IF (ROWCOUNT = 1) then
+                EXECUTE IMMEDIATE 'DELETE FROM IDN_AUTH_USER_SESSION_MAPPING WHERE SESSION_ID IN (SELECT SESSION_ID FROM TEMP_SESSION_BATCH)';
+                sessionmappingcleanupcount := SQL%rowcount;
+                COMMIT;
 
-            IF ( tracingenabled ) THEN
-            EXECUTE IMMEDIATE 'INSERT INTO LOG_WSO2_SESSION_CLEANUP_SP (TIMESTAMP,LOG) VALUES (TO_CHAR( SYSTIMESTAMP, ''DD.MM.YYYY HH24:MI:SS:FF4''),''DELETED USER-SESSION MAPPINGS COMPLETED WITH '|| sessionmappingcleanupcount||''')';
-            COMMIT;
-            END IF;
+                IF ( tracingenabled ) THEN
```
Comment (inthirakumaaran) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/oracle/sessiondata-cleanup/oracle-sessiondata-cleanup.sql:146]: remove the spaces

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-11x/sessiondata-cleanup/postgresql_11-session-data-cleanup.sql [hunk 2]
```diff
@@ -340,21 +389,65 @@ LOOP
...
+        THEN
+            IF (enableLog AND logLevel IN ('TRACE')) THEN
+                notice := 'BATCH DELETE START ON TABLE '||purgingSessionUserMappingTable||' WITH :'||batchCount;
+                RAISE NOTICE '%',notice;
+            END IF;
 
-        EXECUTE 'DELETE from '||quote_ident(purgingSessionUserMappingTable)||' a USING '||purgingBatchTable||' b where a.'||purgeBseColmn||' = b.'||purgeBseColmn||'';
-        GET diagnostics deleteMappingCount := ROW_COUNT;
-        COMMIT;
+            EXECUTE 'DELETE from '||quote_ident(purgingSessionUserMappingTable)||' a USING '||purgingBatchTable||' b where a.'||purgeBseColmn||' = b.'||purgeBseColmn||'';
```
Comment (inthirakumaaran) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-11x/sessiondata-cleanup/postgresql_11-session-data-cleanup.sql:400]: Do we need this space?

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/oracle/sessiondata-cleanup/oracle-sessiondata-cleanup.sql (hunk 1 above)
Comment (inthirakumaaran) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/oracle/sessiondata-cleanup/oracle-sessiondata-cleanup.sql:146]: Check other places as well

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-9x/sessiondata-cleanup/postgresql-session-data-cleanup.sql [hunk 3]
```diff
@@ -344,10 +354,49 @@ auditTable :=  purgingTable||'_auditlog';
 	            GET diagnostics deleteMappingCount := ROW_COUNT;
 
 	            IF (enableLog AND logLevel IN ('DEBUG','TRACE')) THEN
-	            notice := 'BATCH DELETE FINISHED ON TABLE'||purgingSessionUserMappingTable||' WITH :'||deleteMappingCount;
+	            notice := 'BATCH DELETE FINISHED ON TABLE '||purgingSessionUserMappingTable||' WITH :'||deleteMappingCount;
```
Comment (inthirakumaaran) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-9x/sessiondata-cleanup/postgresql-session-data-cleanup.sql:357]: Do we need this change?

Comment (DinikaSen) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-9x/sessiondata-cleanup/postgresql-session-data-cleanup.sql:357]: This is only a formatting fix to add a missing space to the log message

**Review by inthirakumaaran — APPROVED** [review]

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-11x/sessiondata-cleanup/postgresql_11-session-data-cleanup.sql (hunk 2 above)
Comment (DinikaSen) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-11x/sessiondata-cleanup/postgresql_11-session-data-cleanup.sql:398]: Addressed in https://github.com/wso2/carbon-identity-framework/pull/3178/commits/3d2233c7745ad78dc3f6e7e384b163ed21b9ddaa

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/oracle/sessiondata-cleanup/oracle-sessiondata-cleanup.sql (hunk 1 above)
Comment (DinikaSen) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/oracle/sessiondata-cleanup/oracle-sessiondata-cleanup.sql:145]: Addressed in https://github.com/wso2/carbon-identity-framework/pull/3178/commits/3d2233c7745ad78dc3f6e7e384b163ed21b9ddaa

**Review by JKAUSHALYA — APPROVED** [review]

**Pull request merged**